### PR TITLE
[STTNHUB-179] ingest(eventsml): Fix date parsing and add new Event fields

### PR DESCRIPTION
**Pull request by MarkLark86**
Depends on https://github.com/superdesk/superdesk-core/pull/2383

### CODE REVIEW

File: server/planning/feed_parsers/events_ml_test.py [hunk 1]
```diff
@@ -104,5 +110,130 @@ def test_content(self):
...
+                tz=self.app.config["DEFAULT_TIMEZONE"],
+            ),
+        )
+
+        # Only start date & time supplied, with time NOT midnight in local time
+        self.assertEqual(
+            get_item_dates("2022-07-05T18:00:00+03:00"),
+            dict(
+                start=datetime(2022, 7, 5, 15, 0, tzinfo=utc),
+                end=datetime(2022, 7, 5, 16, 0, tzinfo=utc),
```
Comment (petrjasek) [inline server/planning/feed_parsers/events_ml_test.py:162]: I'm not sure if we should interpret this as 1h event, imo would be better to have explicit end=None, once ingested we would loose that information

**Review by petrjasek — APPROVED** [review]

File: server/planning/feed_parsers/events_ml_test.py (hunk 1 above)
Comment (MarkLark86) [reply server/planning/feed_parsers/events_ml_test.py:162]: We have a config in the system for the default Event duration (the system expects start & end dates for all Events).

https://github.com/superdesk/superdesk-planning/blob/develop/server/planning/feed_parsers/events_ml.py#L133
This same config is used in the front end as well as the back end.

Comment (petrjasek) [reply server/planning/feed_parsers/events_ml_test.py:162]: yep but that doesn't really reflect the reality if it's unknown, or do we show in the UI it's using the default and we don't know the actual length?

Comment (MarkLark86) [reply server/planning/feed_parsers/events_ml_test.py:162]: We're always storing an end date

Comment (MarkLark86) [reply server/planning/feed_parsers/events_ml_test.py:162]: It's untested if having end=None will be supported at this time

Comment (petrjasek) [reply server/planning/feed_parsers/events_ml_test.py:162]: ok np, if that's how it works now we should use that

**Pull request merged**